flatlaf-core/src/test/java/com/formdev/flatlaf/ui/TestFlatStyleableInfo.java

```diff
@@ -965,6 +965,7 @@ void tree() {
 			"selectionInactiveBackground", Color.class,
 			"selectionInactiveForeground", Color.class,
 			"selectionBorderColor", Color.class,
+			"alternateRowColor", Color.class,
```

Comment:
Please add `alternateRowColor` also to classes `TestFlatStyling` and `TestFlatStyleableValue`